src/sections/collection/collection-items-panel/useGetAccumulatedItems.tsx

```diff
@@ -58,12 +59,22 @@ export const useGetAccumulatedItems = ({
   ): Promise<number | undefined> => {
     setIsLoadingItems(true)
 
+    const selectedSearchServiceFromSessionStorage: string | null = sessionStorage.getItem(
```

Comment:
since this selectedSearchService is removed from sessionStorage during loadMore, it seems like this would cause a problem if the natural language search contains multiple pages of results. When the next items are loaded, it would revert back to the default search, unless I'm missing something?